data.sql

```diff
@@ -3,4 +3,10 @@
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(1,'Agumon','2020-2-3',0,TRUE,10.23);
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(2,'Gabumon','2018-11-15',2,TRUE,8);
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(3,'Pikachu','2021-1-7',1,FALSE,15.04);
-INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(4,'Devimon','2017-5-12',5,TRUE,11);
\ No newline at end of file
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(4,'Devimon','2017-5-12',5,TRUE,11);
```

Comment:
[OPTIONAL]
- I noticed that you're hardcoding values into the `id` column. What do you think about not having to bother about what number is next to be inserted into the `id` column?(imagine you had ten million rows) 😉
You can add the `GENERATED ALWAYS AS IDENTITY` constraint to the `id` column. This ensures that sequential integers that cannot be altered are generated each time a record is being created. I hope it makes sense. 😄 
You can go through this [article](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.).

- You can also consider adding spaces after the commas `,` to enhance readability. 😉